packages/make-styles/src/index.ts

```diff
@@ -1,7 +1,7 @@
 export { createDOMRenderer } from './renderer/createDOMRenderer';
 
 export { ax } from './ax';
-export { makeStyles } from './makeStyles';
+export { makeStylesCompat } from './makeStyles';
```

Comment:
```suggestion
export { makeStylesCompat } from './makeStylesCompat';
```

Let's also rename a file itself